src/components/Header.js

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
 import { useState } from "react";
+import { product_list } from "./product-list";
+
+function filterData(searchInput,productList) {
```

Comment:
instead of filterData, name it filterProductList, so that the name is more meaningful and readable.